app/Dockerfile

```diff
@@ -25,6 +29,7 @@ RUN mkdir -p /mattermost/data \
 
 # Configure entrypoint and command
 COPY entrypoint.sh /
+RUN chmod +x entrypoint.sh
```

Comment:
Same here.
Should remove this line, run `chmod 755 entrypoint.sh` inside your repository and commit the change :)